Makefile

```diff
@@ -16,4 +16,16 @@ operator-build: operator-sdk
 operator-push:
 	docker push $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG)
 
-.PHONY: operator-build operator-push
+generate: operator-sdk
+	./operator-sdk generate k8s
+	./operator-sdk generate crds
```

Comment:
I think we should add `./operator-sdk generate csv` also

Reply:
`operator-sdk generate csv` requires some parameters such as operator name, description, provider, maintainers' names and emails, etc.. I think we should add it in a new PR.

Reply:
Sounds good, thanks